scripts/db_agent.py

```
@@ -0,0 +1,25 @@
+def troubleshoot_connection(issue: str, model: str = "openllama") -> str:
+    """Use Ollama to get help troubleshooting a connection issue.
+
+    Parameters
+    ----------
+    issue: str
+        Description of the connection problem.
+    model: str
+        Name of the ollama model to use.
+    """
+    try:
+        result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)